snippets.healthcare.Hl7v2MessageTests: test_Hl7v2MessageList failed

This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 33663ccf3c275830ebfdbd740507bce3b6bf614a
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/baa86925-bde6-4cae-ab3f-d4a74bfd6efe), [Sponge](http://sponge2/baa86925-bde6-4cae-ab3f-d4a74bfd6efe)
status: failed
<details><summary>Test output</summary><br><pre>java.lang.AssertionError: 

Expected: a string containing "Retrieved "
     but: was "HL7v2 message created: {
  "createTime" : "1970-01-01T00:00:00Z",
  "data" : "TVNIfF5+XCZ8QXxTRU5EX0ZBQ0lMSVRZfEF8QXwyMDE4MDEwMTAwMDAwMHx8VFlQRV5BfDIwMTgwMTAxMDAwMDAwfFR8MC4wfHx8QUF8fDAwfEFTQ0lJDUVWTnxBMDB8MjAxODAxMDEwMDAwMDANUElEfHwxMTExMTFeXl5eTVJOfDExMTExMTExXl5eXk1STn4xMTExMTExMTExXl5eXk5IU05NQlINCg==",
  "messageType" : "TYPE",
  "name" : "projects/java-docs-samples-testing/locations/us-central1/datasets/dataset-63a9c1f3_7ece_4346_98e1_06ff19e1035c/hl7V2Stores/hl7v2-8e2f3b7c_f679_4369_879f_97b0d3b4b994/messages/W5_pxOBkoLoCxiFxE4cg8zwEWRzMlOzIfeLBrZPf0Zg=",
  "patientIds" : [ {
    "type" : "MRN",
    "value" : "111111"
  }, {
    "type" : "MRN",
    "value" : "11111111"
  }, {
    "type" : "NHSNMBR",
    "value" : "1111111111"
  } ],
  "sendFacility" : "SEND_FACILITY",
  "sendTime" : "2018-01-01T06:00:00Z"
}
"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:964)
	at org.junit.Assert.assertThat(Assert.java:930)
	at snippets.healthcare.Hl7v2MessageTests.test_Hl7v2MessageList(Hl7v2MessageTests.java:144)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
</pre></details>

```
------------------------------------------------------------
- testing healthcare/v1
------------------------------------------------------------
[ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 6.746 s <<< FAILURE! - in snippets.healthcare.Hl7v2MessageTests
[ERROR] snippets.healthcare.Hl7v2MessageTests.test_Hl7v2MessageList  Time elapsed: 0.532 s  <<< FAILURE!
java.lang.AssertionError:

Expected: a string containing "Retrieved "
     but: was ""
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:964)
	at org.junit.Assert.assertThat(Assert.java:930)
	at snippets.healthcare.Hl7v2MessageTests.test_Hl7v2MessageList(Hl7v2MessageTests.java:142)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)

[ERROR] Failures:
[ERROR]   Hl7v2MessageTests.test_Hl7v2MessageList:142
Expected: a string containing "Retrieved "
     but: was ""
[ERROR] Tests run: 57, Failures: 1, Errors: 0, Skipped: 0
```

Also fails in #3251 - either the tests or the service is flakey.  Please figure out which.
```
------------------------------------------------------------
- testing healthcare/v1
------------------------------------------------------------
[ERROR] Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.429 s <<< FAILURE! - in snippets.healthcare.Hl7v2MessageTests
[ERROR] snippets.healthcare.Hl7v2MessageTests.test_Hl7v2MessageList  Time elapsed: 0.526 s  <<< FAILURE!
java.lang.AssertionError:

Expected: a string containing "Retrieved "
     but: was ""
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:964)
	at org.junit.Assert.assertThat(Assert.java:930)
	at snippets.healthcare.Hl7v2MessageTests.test_Hl7v2MessageList(Hl7v2MessageTests.java:142)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)

[ERROR] Failures:
[ERROR]   Hl7v2MessageTests.test_Hl7v2MessageList:142
Expected: a string containing "Retrieved "
     but: was ""
[ERROR] Tests run: 57, Failures: 1, Errors: 0, Skipped: 0
```

--- a/healthcare/v1/src/main/java/snippets/healthcare/hl7v2/messages/HL7v2MessageList.java
+++ b/healthcare/v1/src/main/java/snippets/healthcare/hl7v2/messages/HL7v2MessageList.java
@@ -22,7 +22,6 @@ import com.google.api.client.http.javanet.NetHttpTransport;
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.healthcare.v1.CloudHealthcare;
-import com.google.api.services.healthcare.v1.CloudHealthcare.Projects.Locations.Datasets.Hl7V2Stores.Messages;
 import com.google.api.services.healthcare.v1.CloudHealthcareScopes;
 import com.google.api.services.healthcare.v1.model.ListMessagesResponse;
 import com.google.api.services.healthcare.v1.model.Message;
@@ -98,4 +97,3 @@ public class HL7v2MessageList {
   }
 }
 // [END healthcare_list_hl7v2_messages]
-

--- a/healthcare/v1/src/main/java/snippets/healthcare/hl7v2/messages/HL7v2MessageList.java
+++ b/healthcare/v1/src/main/java/snippets/healthcare/hl7v2/messages/HL7v2MessageList.java
@@ -29,6 +29,7 @@ import com.google.auth.http.HttpCredentialsAdapter;
 import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
 import java.util.Collections;
+import java.util.List;
 
 public class HL7v2MessageList {
   private static final String HL7v2_NAME = "projects/%s/locations/%s/datasets/%s/hl7V2Stores/%s";

--- a/healthcare/v1/src/main/java/snippets/healthcare/hl7v2/messages/HL7v2MessageList.java
+++ b/healthcare/v1/src/main/java/snippets/healthcare/hl7v2/messages/HL7v2MessageList.java
@@ -47,9 +47,11 @@ public class HL7v2MessageList {
     // Results are paginated, so multiple queries may be required.
     String pageToken = null;
 
-    // Create request and configure any parameters.
-    try {
-      Messages.List request =
+    List<Message> hl7v2Messages;
+
+    do {
+      // Create request and execute.
+      ListMessagesResponse messageResponse =
           client
               .projects()
               .locations()
@@ -57,24 +59,20 @@ public class HL7v2MessageList {
               .hl7V2Stores()
               .messages()
               .list(hl7v2StoreName)
-              .setPageSize(100) // Specify pageSize up to 1000
-              .setPageToken(pageToken);
+              .setPageSize(100)
+              .setPageToken(pageToken)
+              .execute();
 
-      ListMessagesResponse response;
-      // Execute response and collect results.
-      do {
-        response = request.execute();
-        if (response.getHl7V2Messages() == null) {
-          continue;
-        }
-        System.out.printf("Retrieved %s HL7v2 messages: \n", response.getHl7V2Messages().size());
-        for (Message message : response.getHl7V2Messages()) {
-          System.out.println("\t" + message);
-        }
-        request.setPageToken(response.getNextPageToken());
-      } while (response.getNextPageToken() != null);
-    } catch (IOException e) {
-      System.out.println("Unable to list HL7v2 messages:" + e.toString());
+      // Collect results.
+      hl7v2Messages = messageResponse.getHl7V2Messages();
+
+      // Update the page token for the next request.
+      pageToken = messageResponse.getNextPageToken();
+    } while (pageToken != null);
+    // Print results.
+    System.out.printf("Retrieved %s HL7v2 messages: \n", hl7v2Messages.size());
+    for (Message message : hl7v2Messages) {
+      System.out.printf("%s\n", message.getName());
     }
   }
 
@@ -100,3 +98,4 @@ public class HL7v2MessageList {
   }
 }
 // [END healthcare_list_hl7v2_messages]
+

